tests/systemtests/upgrade_test.go

```diff
@@ -1,4 +1,4 @@
-//go:build system_test && linux
+//go:build system_test
```

Comment:
Please revert this change. We don't have a historic OSX/ Windows binary to make the test pass on other platforms

Reply:
ah forgot to add that back, thanks

Reply:
added it back